www/_template/getting-started.md

```diff
@@ -3,43 +3,278 @@ layout: layouts/main.njk
 title: Get Started
 ---
 
-## Quick start
+![Webpack vs. Snowpack diagram](/img/snowpack-unbundled-example-3.png)
+
+This guide shows you how to set up Snowpack from scratch in a Node.js project. Along the way you'll learn key concepts of Snowpack and unbundled development
+
+In this guide you'll learn
+
+- Unbundled development: saving time by building only the files changed during development instead of bundling your entire codebase.
+- ESM and Snowpack: Snowpack uses JavaScript's native module system (known as [ESM](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)) to power the single-file builds.
+- Snowpack's dev server: an instant dev environment for unbundled development.
+- Snowpack's build pipeline: a flexible system where you use a bundler only if you need one.
+- Using Node modules/npm packages: Snowpack does the work of bringing your favorite Node modules into the ESM world.
+- Using CSS: Snowpack has built-in support for CSS (and CSS Modules, JSX and [more](/features)
+- Using plugins: you can extend your build even further with [custom plugins](/plugins) that connect Snowpack with your favorite build tools: TypeScript, Babel, Vue, Svelte, PostCSS, Sass... go wild!
+
+> 💡 Tip: the project you'll create here is [Create Snowpack App minimalist template/example](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/). A list other examples is available at the [create-snowpack-app](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/cli) docs.
+
+Prerequisites: Snowpack is a command line tool installed from npm. This guide assumes a basic understanding of Node.js, npm, and how to run commands in the terminal.
+
+Snowpack builds your site for both modern and legacy browsers, but during development you'll need to use a [modern browser](http://caniuse.com/#feat=es6-module). Any recent release of Firefox, Chrome, Safari or Edge supports the modern, bundle-free ESM imports that load your application in the browser.
+
+## Install Snowpack
+
+In this step you'll create a new npm project and install Snowpack.
+
+First create an empty directory. You can use any tool of your choice or the command line as shown here:
 
 ```bash
-npx create-snowpack-app new-dir --template @snowpack/app-template-blank
-cd new-dir
-npm start
+mkdir my-first-snowpack
+cd my-first-snowpack
 ```
 
-This uses [our create-snowpack-app](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/cli) template to spin up a working Snowpack app. For a list of other templates available check out the [create-snowpack-app](https://github.com/snowpackjs/snowpack/tree/master/create-snowpack-app/cli) docs.
+Now enable it as an npm project with the following command, which will create a package.json. Feel free to just hit enter for all the fields:
+
+```bash
+npm init
+```
+
+> 💡 Tip: add the "--use-yarn" or "--use-pnpm" flag to use something other than npm
```

Comment:
Tip: In a hurry? You can run `npm init --yes` to skip the prompts and generate a package.json with npm's default, recommended fields.